rasa/server.py

```diff
@@ -137,6 +137,25 @@ def decorated(*args, **kwargs):
     return decorator
 
 
+def ensure_conversation_exists(app: Sanic):
+    """Wraps a request handler ensuring the conversation exists.
+    """
+
+    def decorator(f):
+        @wraps(f)
+        def decorated(*args, **kwargs):
+            conversation_id = kwargs["conversation_id"]
+            tracker = app.agent.create_processor().get_tracker(conversation_id)
```

Comment:
I'm a bit worried about the performance implications of this as this means that we fetch and de-serialize the tracker twice for these endpoints. What do you think of adding a method `exists` to the `TrackerStore` interface which by default uses `return self.retrieve(conversation_id) is not None` and which we can re-implement with faster versions for each `TrackerStore`?